firmware/libraries/versavis/src/Imu.cpp

```diff
@@ -30,25 +31,31 @@ void Imu::triggerMeasurement() {
 void Imu::publish() {
   if (Sensor::isNewMeasurementAvailable()) {
     DEBUG_PRINTLN((topic_ + " (Imu.cpp): Publish."));
+    bool success;
 #ifdef DEBUG
-    updateData();
+    success = updateDataRecursive(0u, false);

```

Comment:
No differentiation between DEBUG and RELEASE necessary anymore.

Reply:
Not necessarily. I'd like to keep it in anyways to make it easier to switch if there is a problem with the IMU to debug.